imports/plugins/core/accounts/client/templates/profile/userOrdersList.js

```diff
@@ -0,0 +1,46 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Orders } from "/lib/collections";
+import { Reaction } from "/client/api";
+import CompletedOrder from "/imports/plugins/core/accounts/client/containers/userOrdersListContainer";
+
+/**
+ * @method getOrders
+ * @summary returns user's orders
+ * @since 1.5.1
+ * @return {Array} - an array of orders
+ */
+function getOrders() {
```

Comment:
I'm confused as to why we are doing anything at all in the template. If we still need the template at all the only thing it should be doing is rendering the component.